src/components/progress-report/workflow/progress-report-workflow.repository.ts

```diff
@@ -216,7 +173,7 @@ export class ProgressReportWorkflowRepository extends DtoRepository(
       .where({ 'email.value': inArray(emails) })
       .return<{
         id: ID;
-        email: string;
+        email: string | null;
```

Comment:
You can revert this change. We know the emails exist, that's our input/starting point.